docs/learn/about-iota/why-is-iota-feeless.md

```diff
@@ -5,18 +5,45 @@ description: The unique feature of IOTA is the absence of any Fee's in the proto
 
-content
+A blockchain is a ledger published to and tracked by any number of independent participants. Each of these participants, those who track the current state of the ledger, is called a node. The structure of this ledger is a set of blocks of information chained together in chronological order, hence the name "blockchain," where any given block of information contains a header and a body.
+
+Block Header Contents: a 4-byte version number, a 32-byte double hash of the previous block's header, a 32-byte Merkle Root of the transactions in the current block, the 4-byte Unix time when the block was mined, a 4-byte rounded off version of the target difficulty in mining that block, and finally a 4-byte nonce that can be any value set by the miner in order to meet a specific condition we'll discuss more in a moment.
+
+Block Body: Protocol-dependent number of bytes, often 1,000,000 or more, of transaction data.
+
+We mentioned mining several times above. Here we look at what that means for cryptocurrencies built on a proof of work model. When a given block has just been added to the blockchain, a race begins. In this race, to mine the next block, the participants -- most commonly referred to as miners -- gather as many open transactions as they can into a new block body and calculate the Merkle Root from that, and they calculate other information for their new block header as well, all except the 4-byte nonce we mentioned above.
+
+The miner can set this nonce to any value they like, but the goal in doing so is to make the hash of the block header meet the target difficulty. To slightly oversimplify it, the nonce is changed and the header hashed repeatedly until the resulting hash begins with enough zeros. At current difficulty, approximately 1 in every 75,000,000,000,000,000,000,000 possible headers will produce a hash with enough leading zeros to be valid. When one is found, the block is mined and can be sent to the nodes where it is added to the blockchain, and the race begins again for the next block.
+
+This race to mine the next block is a key security feature in blockchain. If one could mine blocks at will, an attacker could make a competing blockchain and, by getting enough nodes to publish it, could create fraudulent transactions, reverse legitimate ones, or any of a number of other things that would rapidly cause trust in the system and the value of the associated coin to trend to 0.
+
+Because each block in the blockchain contains a double hash of the header of the previous block, any change made earlier in the blockchain requires all subsequent blocks be changed also, and these changes require the newly-changed blocks be mined again. It is easy to see that this system provides security not only for new transactions, but that the older a transaction in the blockchain is, the more secure it becomes.
```

Comment:
Because each block in the blockchain contains a double hash of the header of the previous block, any change made earlier in the blockchain requires all subsequent blocks to be changed also, and these changes require the newly-changed blocks to be mined again. It is easy to see that this system provides security not only for new transactions, but that the older a transaction in the blockchain is, the more secure it becomes.